components/ipfs/ipfs_file_import_worker.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_IPFS_IPFS_FILE_IMPORT_WORKER_H_
+#define BRAVE_COMPONENTS_IPFS_IPFS_FILE_IMPORT_WORKER_H_
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "base/callback.h"
+#include "brave/components/ipfs/imported_data.h"
+#include "brave/components/ipfs/ipfs_import_worker_base.h"
+#include "url/gurl.h"
+
+namespace ipfs {
+
+class IpfsFileImportWorker : public IpfsImportWorkerBase {
+ public:
+  IpfsFileImportWorker(content::BrowserContext* context,
+                       const GURL& endpoint,
+                       ImportCompletedCallback callback,
+                       const base::FilePath& path);
+  ~IpfsFileImportWorker() override;
```

Comment:
Sorry if I wasn't clear enough, when I say DISALLOW_COPY_AND_ASSIGN is deprecated, I expect explicit defines below being added, not just remove that line.
```
IpfsFileImportWorker(const IpfsFileImportWorker&) = delete;
IpfsFileImportWorker& operator=(const IpfsFileImportWorker&) = delete;
```
These are also missing in IpfsLinkImportWorker and IpfsTextImportWorker, which I was expected them to be added too in the previous PR, but I probably missed to double check these last time.